### DISCUSSION TITLE
How can I make a functional program with it

### DISCUSSION THREAD

**Discussion by Aesistril** [ParthJadhav/Tkinter-Designer#103] [Q&A] [answered]
I looked up readme.md but there is no information about running code with buttons and sending input to the code. How can I do this? I will add it to readme if someone teaches me

**Comment by ParthJadhav** [comment]
I am not getting what are you meaning to tell. Please explain a bit more.

**Comment by leeso2021** [comment, marked as answer]
I suggest u taking a look at gui.py u'll find the answer

**Comment by Aesistril** [comment]
Oh sorry
